agent.go

```diff
@@ -73,7 +73,7 @@ func (a *Agent) ListenAndServe() {
 	// server needs to allow redirections to localhost:7773/oauth2/callback
 	// 7773 is chosen by looking wiresteward initials hex on ascii table
 	// (w = 0x77 and s = 0x73)
-	if err := http.ListenAndServe("127.0.0.1:7773", nil); err != nil {
+	if err := http.ListenAndServe(*flagAgentAddress, nil); err != nil {
```

Comment:
the comment on the above line should be moved next to the default flag value now?